MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -49,6 +49,16 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var maneuverDirection: ManeuverDirection
     
+    /**
+     An abbreviated version of the text for a given component.
+     */
+    @objc public var abbreviation: String?
+    
+    /**
+     The priority in which the component should be abbreviated. Lower numbers should be abbreviated first.
+     */
+    @objc public var abbreviationPriority: Int
```

Comment:
Although this property is optional, objc does not support optional Ints. Because of this, the default value will be zero. Anyone have objections here?

Reply:
No concerns from me. Just noting that `abbr_priority: 0` is the highest priority, so you might have to add a check to excludes abbreviations where `abbr` is undefined when putting abbreviations together.

Reply:
On second thought, I think it's important it is optional. Going to use NSNumber.

Reply:
Changed the default value to `Int.max` since zero would be incorrect.